.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -7,14 +7,15 @@ on:
     types: [published]
 
 env:
-  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+  CIBW_BUILD: 'cp37-* cp38-* cp39-* cp310-*'
+  CIBW_SKIP: "*-musllinux*"
```

Comment:
For my understanding: what is being skipped here and for what reason?

Reply:
We skip attempting to build MUSL supported wheels (an alternative to the more common glibc linux distributions). This can be enabled a little later when MUSL support is more further along.

Reply:
@antalszava Alpine Linux is typically used in Docker containers, so this is where MUSL compatibility is usually required

Reply:
Thank you!